packages/dart.rb

```diff
@@ -39,5 +39,6 @@ def self.install
     system "cp -r bin/ #{CREW_DEST_PREFIX}"
     system "cp -r lib/* #{CREW_DEST_LIB_PREFIX}"
     system "cp -r include/ #{CREW_DEST_PREFIX}"
+    system "cp version /usr/local/version" # This stops 'pub get' from throwing errors
```

Comment:
Thank you for the update @Ewpratten!  Can we use a crew constant to make sure this file is tracked?
```
system "cp version #{CREW_DEST_PREFIX}/version"
```

Reply:
Done!